app/src/main/java/com/aleksagn/playlistmaker/data/network/ITunesApiService.kt

```diff
@@ -1,13 +1,10 @@
 package com.aleksagn.playlistmaker.data.network
 
 import com.aleksagn.playlistmaker.data.dto.TracksSearchResponse
-import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Query
 
 interface ITunesApiService {
     @GET("/search")
```

Comment:
Также стоит добавить `entity`:
```suggestion
    @GET("/search?entity=song")
```
Тогда только валидные треки будут приходит в ответе

Reply:
Выполнено